modules/apps/layout/layout-page-template-service/src/main/java/com/liferay/layout/page/template/internal/upgrade/v1_1_0/UpgradeLayoutPrototype.java

```diff
@@ -97,14 +100,28 @@ protected void upgradeLayoutPrototype() throws Exception {
 					int i = 1;
 
 					while (true) {
-						String newName = name + StringPool.DASH + i;
+						String newName;
+
+						if (name.length() == 75) {
+							newName = name.substring(0, 73);
+						}
+						else {
+							newName = name;
+						}
+
+						newName = newName + StringPool.DASH + i;
 
 						if (existingNames.contains(newName)) {
 							i++;
 
 							continue;
 						}
 
+						_log.error(
```

Comment:
@SamZiemer It should be a warning and we need to use:
if (_log.isWarnEnabled()) {

Thanks.